src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,91 @@
+import PropTypes from 'prop-types';
 import React, { Component } from 'react';
+import { Field } from '../Field/Field';
+import { names, initialState, indexes, status, reg } from '../../api/constants';
+
+const allValid = {
```

Comment:
давай более конкретные названия
```suggestion
const areFieldsValid = {
```